auditwheel/policy/__init__.py

```diff
@@ -17,8 +17,10 @@
 _PLATFORM_REPLACEMENT_MAP = {
     'manylinux1_x86_64': ['linux_x86_64'],
     'manylinux2010_x86_64': ['linux_x86_64'],
+    'manylinux2014_x86_64': ['linux_x86_64'],
     'manylinux1_i686': ['linux_i686'],
     'manylinux2010_i686': ['linux_i686'],
+    'manylinux2014_i686': ['linux_i686'],
```

Comment:
Similar to https://github.com/pypa/warehouse/pull/6684, we also need to add the other architectures that PEP 599 supports here.

Reply:
I'd like to get this done in a second time. There's been literally no tests on those other platforms so unless we say that support for other architectures is experimental for now, I'd rather do it in 2 times.